geomstats/hypersphere.py

```diff
@@ -134,6 +134,25 @@ def random_uniform(self, n_samples=1):
         point = self.intrinsic_to_extrinsic_coords(point)
 
         return point
+      
+    def random_vonMisesFisher(self, K=10, n_samples=1):
+        """
+        Sample in the 2-sphere with the von Mises distribution
+        centered in the north pole
+        """
+        if self.dimension != 2:
+            raise NotImplementedError(
+                    'Sampling from the von Mises Fisher distribution'
+                    'is only implemented in dimension 2')
+        u = 2 * gs.pi * gs.random.rand(n_samples)
+        v = gs.vstack((gs.cos(u), gs.sin(u)))
+        xi = gs.random.rand(n_samples)
+        W = 1 + 1/K * gs.log(xi + (1-xi) * gs.exp(-2 * K))
```

Comment:
In the new pull request (https://github.com/geomstats/geomstats/pull/129), I have tried to be compliant with the PEP8 guidelines, in particular I have added whitespace around the operators of lowest priority : https://www.python.org/dev/peps/pep-0008/#other-recommendations.

I use the editor Anaconda-Spyder, and I have enabled the code analyzer, but it does not correct this. In case flake8 is more precise with spacing, I have installed it but I have not managed to use it on Spyder. If the style of the new pull request is still an issue, I could use some help ! Thanks a lot !